orc8r/cloud/go/services/orchestrator/servicers/builder_servicer_test.go

```diff
@@ -32,6 +33,93 @@ import (
 	mconfig_protos "magma/orc8r/lib/go/protos/mconfig"
 )
 
+func testUtilForTestSharedConfig(t *testing.T, orc8rVersion string) {
```

Comment:
Nit here and below

- The "shared config" points more to the `shared_mconfig` rather than the magmad mconfig
- Fn names to generally describe "what" not "why", and be verbs rather than nouns

So maybe something like `testMagmadOrc8rVersion`

Reply:
this is a util function function, for the test function testing shared config, and not the actual test itself, so i am not sure if the above is applicable.

Reply:
Let me try to clarify -- 1. "shared config" makes me think of the shared_mconfig, meaning it's a name overload, meaning it's productive to choose a different name that doesn't cause the semantic overload

2. See https://github.com/magma/magma/wiki/Contributing-Code-Conventions#style for verbifying function names